### PR TITLE
Code examples for episode 055 & 056

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: sample_code/ep055/rev07/t.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```
Comment (int3l) [inline sample_code/ep055/rev07/t.py:1]: Added additional examples about other things that can go before the `__future__` import like the shebang and the magic coding comment.

Comment (asottile) [reply sample_code/ep055/rev07/t.py:1]: idk if the coding comment is worth mentioning, since it's basically obsolete

Comment (int3l) [reply sample_code/ep055/rev07/t.py:1]: Yeah, what I found in the docs is that it's still supported, so you can deviate from the default utf-8 encoding if needed. Maybe I can change the `utf-8` value to something else, or do you prefer to remove that line?

Comment (asottile) [reply sample_code/ep055/rev07/t.py:1]: I'd prefer to remove it, you an use any old comment though -- that's also allowed before

**Review by asottile — APPROVED** [review]

**Pull request merged**